docs/tutorial/filter.md

```diff
@@ -1,5 +1,7 @@
 # Running Filter Protocol
 
+> TODO: Fix or depricate
+
```

Comment:
No strong opinion here, but perhaps we can just add the date and a note to say that this refers to an older version of nwaku and direct people to the operator guide? I think we may want to keep the record of old testnets and tutorials, though I'm open to just deleting these from the repo as well.